app/questionnaire_state/relationship_state_question.py

```diff
@@ -11,13 +11,21 @@ def __init__(self, id, schema_item):
     def build_repeating_state(self, user_input):
         template_answer = self.answers.pop()
         group_instance = template_answer.group_instance
-        household_answers = get_answer_store(current_user).filter(answer_id='household-full-name')
-        sorted(household_answers, key=lambda household_answer: household_answer['answer_instance'])
-        remaining_answers = household_answers[group_instance + 1:] if self.group_instance < len(household_answers) else []
+        first_names = get_answer_store(current_user).filter(answer_id='first-name')
```

Comment:
@benjefferies Discussed with @ajmaddaford. Agreed this should only show first and last names for now.